helpers/arguments.js

```diff
@@ -0,0 +1,23 @@
+
+module.exports.MODULE = '--module';
+
+/**
+ * Check to make sure we have an element name
+ */
+module.exports.ArgumentsHaveElementName = () => {
+
+    // Check if element name exists
+    if(process.argv[2] === undefined) {
```

Comment:
Again, can we kinda clean up the `process.argv[2]` and `[3]` references? It might make more sense to pass them through from the outside code. Just makes for less fragile code imo.